src/components/Reports/SubscriberHistory/SubscriberHistory.js

```diff
@@ -117,7 +133,8 @@ const SubscriberHistory = ({
       </header>
       <section className="dp-container">
         <div className="dp-rowflex">
-          <div className="col-sm-12 m-b-36">
+          <Tabs tabsProperties={state.tabsProperties}></Tabs>
```

Comment:
Maybe you can create a component with these specific tabs and reuse it in these pages

Reply:
I mean, a new layer over these ones.

Reply:
I agree here with Andres. A nice refactor to avoid repeating. Here remeber about orthogonality, what happens if some day I want to add a third tab? I will need this added in both components...

Reply:
I tried to make the component reusable. At least, I have to find a way to send which tab is selected.